Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -76,6 +82,9 @@
         Heat: 0.9
   - type: ExplosionResistance
     damageCoefficient: 0.80
+  - type: TemperatureProtection # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)